test/unit/org/apache/cassandra/index/SecondaryIndexManagerTest.java

```diff
@@ -194,10 +194,12 @@ public void testIndexRebuildWhenAddingSStableViaRemoteReload()
         assertEmpty(execute("SELECT * FROM %s WHERE a=1"));
         assertEmpty(execute("SELECT * FROM %s WHERE c=1"));
 
```

Comment:
This test `testIndexRebuildWhenAddingSStableViaRemoteReload` was added to test legacy writer behavior that built index on writer.  Now writer only builds index on flush, not reload.


During sstable reload, reloaded sstable should have index attached (compaction task would abort if failed to build index; if initial index build is not completed, index is not-queryable, it's fine); unless it's added as shallow sstable.

Reply:
this test can be removed if you prefer